docs/mvp_morevp_interaction.md

```diff
@@ -60,11 +60,22 @@ In each we explain possible ways of preventing the double spend.
 10. Mallory submits IFE -> Mallory piggybacks output -> Mallory waits for finalization -> Mallory does SE on output  
     ! same as (9.)
 
-### Exploring `standardExitId(hash(tx), oindex)`
+### Exploring `standardExitId(txbytes, utxo_pos)`
 Many of solutions described above are possible only if there is a way to have one-to-one mapping between in-flight exits and standard exits.
-To achieve that, we change the way `inFlightExitId` and `standardExitId` are computed, so both are a function of hash(tx).
+To achieve that, we change the way `inFlightExitId` and `standardExitId` are computed, so both are a function of hash(tx) ([see](#Distinction-between-deposit-and-regular-transactions)). 
 This way whenever one type of exit comes in we can check if other type of exit was performed.
 
+
+#### Distinction between deposit and regular transactions
+In case the tx is a deposit, the `standardExitId` is a function of hash(tx concat tx_pos). 
+This distinction was made as when one deposits the same amount twice (or more) the tx_bytes are the same, so exit ids is not unique for the similar deposits, 
```

Comment:
I would like to emphasize "without tx_pos"
eg. `....the tx_bytes are the same, so without concatenating tx_pos, the exit ids would be the same and not unique for the similar deposits...`

Also, I would link the issue to this paragraph as well as there has some valuable discussion on why we chosen this design: https://github.com/omisego/plasma-contracts/issues/80